src/reducers/MapReducer.js

```diff
@@ -8,6 +8,36 @@ import {
 
 import {map} from "../constants";
 
+const sampleSiteMarkers = [
+    {
+        id: 1,
+        title: 'Twin Lakes Hidden Spot',
+        description: 'Beautiful view of Twin Lakes off this hidden forest road.',
+        coordinate: {
+            longitude: -106.391015,
+            latitude: 39.085855
+        }
+    },
+    {
+        id: 2,
+        title: 'Lily Lake',
+        description: 'Nice view of the lilypads in this secluded spot, but a pretty tough road to reach it.',
+        coordinate: {
+            longitude: -106.368051,
+            latitude: 39.351661
+        }
+    },
+    {
+        id: 3,
+        title: 'Slide Lake',
+        description: 'Pretty riverside camping, but a REALLY nasty road to get there.',
+        coordinate: {
+            longitude: -106.389204,
+            latitude: 39.372171
+        }
+    }
+];
+
```

Comment:
Added sample sites above to populate the `MapView` until we have real data coming back from firebase.